proposed/http-factory/http-factory.md

```diff
@@ -210,6 +209,25 @@ interface UploadedFileFactoryInterface
         $clientFilename = null,
         $clientMediaType = null
     );
+
+    /**
+     * Create a new uploaded file from an existing stream.
+     *
+     * The uploaded file SHOULD report a size equal to the length of the stream.
+     *
+     * The uploaded file SHOULD have an error status of `UPLOAD_ERR_OK`.
```

Comment:
@weierophinney is this what you were thinking would be constraints associated with creating uploaded files from streams?